src/components/table/Table.js

```diff
@@ -967,93 +969,95 @@ class Table extends Component {
 
     this.rowRefs = {};
 
-    return rows
-      .filter(row => {
-        if (collapsedRows) {
-          return collapsedRows.indexOf(row[keyProperty]) === -1;
+    let renderRows = rows.filter(row => {
+      if (collapsedRows) {
+        return collapsedRows.indexOf(row[keyProperty]) === -1;
+      }
+      return true;
+    });
+
+    if (isMobileOrTablet && rows.length > MOBILE_TABLE_SIZE_LIMIT) {
+      renderRows = renderRows.slice(0, MOBILE_TABLE_SIZE_LIMIT);
```

Comment:
So how do you switch to the next page if pagination is hidden on mobile devices ?

Reply:
my suggestion, as a quick&dirty solution, was to limit the query to a reasonable number of rows.
If that amount is hit, we display the "limit" message at the bottom of the page.
User has to filter and search if he needs more.
In this way we make the window usable with small amount of effort.

----
THE solution: live scroll or some pagination which looks nice on mobile... but I think live scroll is THE shit!

Reply:
Yeah, we have an issue to implement that. The only problem is when we're showing nested data.